Leno_beta1.5/src/Run1D.cpp

```diff
@@ -121,31 +121,7 @@ void Run1D::runPoisson1D(int argc, char** argv){
 						int num = data.bASemiOnly();
 						int m = 0;
 						int n = 1;
-						//std::cout<<"total number is:"<<num<<endl;
-						//for (int i=0; i<num; i++) {
-							//std::cout<<"Semi phin of ["<<i<<"] layer is:"<<data.phin[i]<<endl;
-							//std::cout<<"Semi phip of ["<<i<<"] layer is:"<<data.phip[i]<<endl;
-						//	std::cout<<"Semi cBSemi of ["<<i<<"] layer is:"<<data.cBSemi[i]<<endl;
-						//	std::cout<<"Semi vBSemi of ["<<i<<"] layer is:"<<data.vBSemi[i]<<endl;
-						//	std::cout<<"The "<<i<<"th material :"<<endl;
-							//std::cout<<"getTAArray: "<<dev1D.getTArray()<<endl;
-						//	vector<Material> vec = dev1D.getMaterialList();
-							//std::cout<<"getMaterialList.index:  "<<vec[i].index<<endl;
-							//std::cout<<"getMaterialList.type:  "<<vec[i].type<<endl;
-						//	std::cout<<"getMaterialList.name:  "<<vec[i].name<<endl;
-						//	std::cout<<"fLn of ["<<i<<"] layer is: "<<data.fLnSemi[i]<<endl;
-						//	std::cout<<"fLp of ["<<i<<"] layer is: "<<data.fLpSemi[i]<<endl;
-						//}
 						for (int i=0; i<num; i++) {
-							//std::cout<<"Semi phin of ["<<i<<"] layer is:"<<data.phin[i]<<endl;
-							//std::cout<<"Semi phip of ["<<i<<"] layer is:"<<data.phip[i]<<endl;
-							//std::cout<<"Semi cBSemi of ["<<i<<"] layer is:"<<data.cBSemi[i]<<endl;
-							//std::cout<<"Semi vBSemi of ["<<i<<"] layer is:"<<data.vBSemi[i]<<endl;	
-						//	double judgecB = data.cBSemi[i]-data.cBSemi[i+1];
-						//	double judgevB = data.vBSemi[i]-data.vBSemi[i+1];
-						//	std::cout<<"judgecB = "<<judgecB<<endl;
-						//	std::cout<<"judgevB = "<<judgevB<<endl;
-						//	if ((abs(judgecB) > 0.5) || (abs(judgevB) > 0.5 )) {
 							if (data.fLnSemi[i] != data.fLnSemi[i+1]) {
 								m=i;
 								n=i+1;
```

Comment:
Add TODO here to remind the future user the flaky cases of this code 1) Vds == 0, 2) If semiconductor is used as the tunneling barrier.